http-client/ingredients-database_utils.ts

```diff
@@ -0,0 +1,72 @@
+import {renderProduct} from '../products_utils'
+import {ingredientData} from '../type/ingredient-data'
+
+export const getAllIngredients = async (ingredientsArray: Array<ingredientData>) => {
+    const loader = $('<div>').addClass('products__loader').appendTo($('.products'))
+    const url = 'http://localhost:3001/ingredients'
+
+    await fetch(url, {
+        method: 'GET'
+    })
+        .then(async res => {
+            ingredientsArray = ingredientsArray.concat(JSON.parse(await res.text()))
+        })
+        .catch(error => {
```

Comment:
it will never go to the catch. I showed how to resolve it this morning on the whiteboard